moderation.go

```diff
@@ -0,0 +1,50 @@
+package helix
+
+// Ban ...
+// ExpiresAt must be parsed manually since an empty string means perma ban
+type Ban struct {
+	UserID    string `json:"user_id"`
+	UserName  string `json:"user_name"`
+	ExpiresAt string `json:"expires_at"`
```

Comment:
We have support for unmarshalling timestamps to `time.Time` values. Would you mind changing this type from `string` to `time.Time`? See the Stream struct for an example: https://github.com/nicklaw5/helix/blob/c9cfaa2ec39b467599a43368176732453ef91a2c/streams.go#L15

Reply:
I'd do that but if a user is permanently banned `expires_at` will be an empty string.

Reply:
Ah k. I was not aware of that.

Reply:
In that case let's use the custom `Time` struct that I introduced for this specific scenario:
https://github.com/nicklaw5/helix/blob/c9cfaa2ec39b467599a43368176732453ef91a2c/time.go#L16-L37